src/stactools/overture_maps/metadata.py

```diff
@@ -0,0 +1,446 @@
+    "For more information, visit the [Overture Maps Foundation](https://overturemaps.org/).\n\n"
+)
+
+
+class PyarrowArgs(TypedDict):
+    where: str
+    filesystem: Optional[Union[S3FileSystem, AzureBlobFileSystem]]
+
+
+class Theme(str, Enum):
+    ADDRESSES = "addresses"
+    BASE = "base"
+    BUILDINGS = "buildings"
+    DIVISIONS = "divisions"
+    PLACES = "places"
+    TRANSPORTATION = "transportation"
+
+
+class FeatureType(Enum):
+    ADDRESS = ("address", Theme.ADDRESSES)
+    BUILDING = ("building", Theme.BUILDINGS)
+    BUILDING_PART = ("building_part", Theme.BUILDINGS)
+    DIVISION = ("division", Theme.DIVISIONS)
+    DIVISION_AREA = ("division_area", Theme.DIVISIONS)
+    DIVISION_BOUNDARY = ("division_boundary", Theme.DIVISIONS)
+    PLACE = ("place", Theme.PLACES)
+    SEGMENT = ("segment", Theme.TRANSPORTATION)
+    CONNECTOR = ("connector", Theme.TRANSPORTATION)
+    BATHYMETRY = ("bathymetry", Theme.BASE)
+    INFRASTRUCTURE = ("infrastructure", Theme.BASE)
+    LAND = ("land", Theme.BASE)
+    LAND_COVER = ("land_cover", Theme.BASE)
+    LAND_USE = ("land_use", Theme.BASE)
+    WATER = ("water", Theme.BASE)
+
+    def __init__(self, type_name: str, theme: Theme):
+        self.type_name = type_name
+        self.theme = theme
+
+    @classmethod
+    def from_string(cls, type_name: str) -> "FeatureType":
+        """Get FeatureType enum from string name"""
+        for feature_type in cls:
+            if feature_type.type_name == type_name:
+                return feature_type
+        raise ValueError(f"Unknown feature type: {type_name}")
+
+    @classmethod
+    def get_types_for_theme(cls, theme: Theme) -> list["FeatureType"]:
+        """Get all feature types for a given theme"""
+        return [ft for ft in cls if ft.theme == theme]
+
+
+class StorageBackend(str, Enum):
+    AWS = "aws"
+    AZURE = "azure"
+    LOCAL = "local"
+
+
+CLOUD_PROVIDERS = {
+    StorageBackend.AWS: Provider(
+        name="Amazon Web Services",
+        roles=[ProviderRole.PRODUCER],
+        url="https://aws.amazon.com",
+    ),
+    StorageBackend.AZURE: Provider(
+        name="Microsoft",
+        roles=[ProviderRole.HOST],
+        url="https://microsoft.com",
+    ),
+}
+
+THEME_LICENSES = {
+    Theme.ADDRESSES: {
+        "href": "https://docs.overturemaps.org/attribution/#addresses",
+        "title": "various",
+        "type": "text/html",
+    },
+    Theme.PLACES: {
+        "href": "https://cdla.dev/permissive-2-0/",
+        "title": "CDLA-Permissive-2.0",
+        "type": "text/html",
+    },
+}
+
+
+@dataclass
+class CollectionInfo:
+    storage_backend: StorageBackend
+    theme: Theme
+    latest_release: str
+
+    @property
+    def theme_description(self) -> str:
+        """Scrape theme description from OvertureMaps/docs repo"""
+        # Get the raw content
+        raw_url = f"https://raw.githubusercontent.com/OvertureMaps/docs/refs/heads/main/docs/guides/{self.theme.value}.mdx"
```

Comment:
If this is `mdx`, there may be some elements that don't render correctly in standard markdown.